src/fidesops/api/v1/endpoints/user_endpoints.py

```diff
@@ -89,6 +96,75 @@ def create_user(
     return user
 
 
+def _validate_current_user(user_id: str, user_from_token: FidesopsUser) -> bool:
```

Comment:
This is a minor nit. It looks like the 2 functions that call `_validate_current_user` don't use the returned `True` value.  The `return True` statement could safely be removed. I could see future code using the return boolean in the future though. 
```suggestion
def _validate_current_user(user_id: str, user_from_token: FidesopsUser) -> None:
```